gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -151,16 +154,19 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
             client_options (ClientOptions): Custom options for the client. It
                 won't take effect unless ``transport`` is None.
                 (1) The ``api_endpoint`` property can be used to override the
-                default endpoint provided by the client. GOOGLE_API_USE_MTLS
+                default endpoint provided by the client. GOOGLE_API_USE_MTLS_ENDPOINT
```

Comment:
This is the new standard https://google.aip.dev/auth/4114. Standard keeps changing in the alpha stage, it should be fine since no one uses mtls. Same thing will happen to Go clients as well. 

@busunkim96

Reply:
I'm slightly more concerned about breaking changes to the Transport constructor. If no one was using mTLS (and this env var) it seems alright to take the rename.

Reply:
Yes, no one is using mTLS and the env var now.